README.md

```diff
@@ -9,6 +9,10 @@ An apt-based package manager
 
 ## Role Variables
 
+| Name         | Required/Default | Description                                                    |
+|:-------------|------------------|----------------------------------------------------------------|
+| `nginx_full` | `false`          | Install the `nginx-full` package or the normal `nginx` package |
```

Comment:
*Whether to install the `nginx-full` package instead of the normal `nginx` package